src/forms/modelForm.tsx

```diff
@@ -124,6 +124,10 @@ class ModelForm extends React.Component<ModelFormProps, ModelFormState> {
       let num_network_ports = field["num_network_ports"];
       console.log(num_network_ports, network_ports);
       let index = network_ports.length;
+      if (!isNaN(num_network_ports) && num_network_ports < 0) {
+        num_network_ports = 0;
+      }
```

Comment:
will this result in the `num_network_ports` field being sometimes `0` and sometimes `null` (in the DB and also in the table wherever we display it? not sure if that is ideal, but not a dealbreaker, just something to track